Camera use is optional, don't disallow install on camera-less devices.
See right side pane at
http://developer.android.com/guide/topics/manifest/uses-permission-element.html

diff --git a/TessUI/AndroidManifest.xml b/TessUI/AndroidManifest.xml
index c94e45f..4147838 100644
--- a/TessUI/AndroidManifest.xml
+++ b/TessUI/AndroidManifest.xml
@@ -6,6 +6,7 @@
     <uses-sdk android:minSdkVersion="8" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-feature android:name="android.hardware.camera" android:required="false" />
     <application android:icon="@drawable/logo" android:label="@string/app_name" android:debuggable="false">
         <activity android:name="org.atai.TessUI.TessUIActivity"
                   android:label="@string/app_name"
